src/accountlist/accountlistSelectors.js

```diff
@@ -0,0 +1,22 @@
+import { fromJS } from 'immutable';
+
+export const getActiveAccount = (state) =>
+  state.accountlist.get(0);
+
+export const getAuth = (state) => {
+  const account = getActiveAccount(state);
+
+  if (!account) {
+    return fromJS({
```

Comment:
Why use `fromJS` instead of explicit constructors (such as `Immutable.Record` or `Immutable.Map`)?